Contuity/Views/Jot/Write/WriteJotViewController.swift

```diff
@@ -37,15 +37,20 @@ class WriteJotViewController: UIViewController {
     /// When the save button is tapped, this is triggered to notify the user they are about to save
     /// A jot with new initiatives attached.
     @objc private func processTextToSave() {
-        let hasNewInitiatives = presenter.update
-            ? Set((Jot.read(givenID: presenter.jotID)?.data ?? "").taggedWords)
-                .isSubset(of: presenter.text.taggedWords)
-            : presenter.text.taggedWords.count > 0
-
-        if hasNewInitiatives {
-            showNewInitiativesAlert()
-        } else {
-            saveTextAsJot()
+        do {
+            let hasNewInitiatives = presenter.update
+                ? Set((try Jot.read(givenID: presenter.jotID).data).taggedWords)
+                    .isSubset(of: presenter.text.taggedWords)
+                : presenter.text.taggedWords.count > 0
+            
+            if hasNewInitiatives {
+                showNewInitiativesAlert()
+            } else {
+                saveTextAsJot()
+            }
+        }
+        catch {
+            
```

Comment:
Why is this change any better? Personally, I dislike using try/catch if the catch is empty. I think the old logic is clearer to read

Reply:
Similar to above read throws an error now so it needs to be in a try catch. I just wasn't sure what to put in the catch. Maybe there's a better way to do it though

Reply:
You should be able to still use `try?` before the read call instead of `try` then coalesce it to []